core/templates/dev/head/editor/ExplorationPreview.js

```diff
@@ -18,19 +18,29 @@
  */
 
 oppia.controller('ExplorationPreview', [
-  '$scope', '$timeout', 'LearnerParamsService', 'explorationData',
+  '$scope', '$timeout', 'LearnerParamsService',
+  'explorationData', 'editorContextService',
   'explorationStatesService', 'explorationInitStateNameService',
   'explorationParamSpecsService', 'explorationTitleService',
   'explorationCategoryService', 'explorationParamChangesService',
   'explorationGadgetsService', 'oppiaPlayerService',
   function(
-      $scope, $timeout, LearnerParamsService, explorationData,
+      $scope, $timeout, LearnerParamsService,
+      explorationData, editorContextService,
       explorationStatesService, explorationInitStateNameService,
       explorationParamSpecsService, explorationTitleService,
       explorationCategoryService, explorationParamChangesService,
       explorationGadgetsService, oppiaPlayerService) {
     $scope.isExplorationPopulated = false;
-    explorationData.getData().then(function() {
+    explorationData.getData(ignoreCache = true).then(function() {
```

Comment:
It isn't needed.
I managed to convince myself that it was before the last commit... I think I must have added the parameter at the same time as fixing a different bug.  Sorry for the confusion!